src/Suggestions/Suggestions.module.css

```diff
@@ -0,0 +1,111 @@
+.suggestionSection {
+  padding: 112px 0;
+  margin: 0 auto;
+  max-width: 600px;
+  width: 100%;
+  box-sizing: border-box;
+}
+.suggestionSection.onHomePage {
+  position: fixed;
+  top: 9%;
+  left: 66%;
+  height: 100vh;
+  width: 293px;
+}
+.suggestedText {
+  font-size: 16px;
+  font-weight: 600;
+}
+.suggestionText.onHomePage {
+  color: #8e8e8e;
+  font-weight: 600;
+}
+.SuggestionTextViewButton {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.seeAllButton {
+  border: 0;
+  background: none;
+  font-weight: 600;
+  color: #262626;
+  padding: 0;
+  margin-right: 12px;
+  cursor: pointer;
+  text-decoration: none;
+  font-size: 12px;
+}
+.suggestionContentSection {
+  background-color: #fff;
+}
+.suggestionContentSection.onHomePage {
+  background-color: transparent;
+}
+.suggestionContent {
+  display: flex;
+  padding: 8px 16px;
+  align-items: center;
+}
+.suggestionContent.onHomePage {
+  padding: 0;
+  box-sizing: border-box;
+  margin-bottom: 12px;
+}
+.suggestionsUserNameReason {
+  flex: 1;
+}
+
+.suggestionUserName {
+  margin: 0;
+  margin-left: 12px;
+  font-weight: 600;
+  text-decoration: none;
+  color: #262626;
+}
+.suggestionUserFullName {
+  margin: 0;
+  margin-left: 12px;
+  margin-top: 2px;
+  color: #8e8e8e;
+  font-weight: 400;
+  font-size: 12px;
+  max-width: 195px;
+}
+.suggestionUserReason {
+  margin: 0;
+  margin-left: 12px;
+
+  padding-top: 2px;
+  color: #8e8e8e;
+  font-weight: 400;
+  font-size: 12px;
+}
+.suggestionUserReason.onHomePage {
+  margin-left: 12px;
+  margin-top: 0px;
+  color: #8e8e8e;
+  font-weight: 400;
+  font-size: 12px;
+  max-width: 185px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+.followButton {
+  border: 1px solid transparent;
+  width: 61px;
+  height: 30px;
+  border-radius: 4px;
+  color: #fff;
+  background-color: #0095f6;
+  font-weight: 600;
+}
+.followButton.onHomePage {
+  border: 0;
+  background: 0;
+  color: #0095f6;
+  padding: 0;
+  cursor: pointer;
+  font-size: 12px;
+}
```

Comment:
Since this is a CSS module and called `Suggestions.module.css`. We don't need the prefix `Suggestion` for every class.

Reply:
On it